src/map/script.cpp

```diff
@@ -26252,6 +26252,36 @@ BUILDIN_FUNC(item_reform){
 #endif
 }
 
+BUILDIN_FUNC(item_enchant){
+#if PACKETVER_RE_NUM < 20211103
+	ShowError( "buildin_item_enchant: This command requires packet version 2021-11-03 or newer.\n" );
+	return SCRIPT_CMD_FAILURE;
+#else
+	struct map_session_data* sd;
+
+	if( !script_charid2sd( 3, sd ) ){
+		return SCRIPT_CMD_FAILURE;
+	}
+
+	// Hardcoded clientside check
+	if( sd->weight >= ( ( sd->max_weight * 70 ) / 100 ) ){
+		ShowError( "buildin_item_enchant: Player %s (AID: %u, CID: %u) was over 70% weight.\n", sd->status.name, sd->status.account_id, sd->status.char_id );
+		return SCRIPT_CMD_FAILURE;
```

Comment:
It does not seem necessary to throw a message on the console since the player is informed by the client, no? the command might just silently fail.
(silent failure will also reduce the amount of checkweight required in item script and npc script)

Reply:
We can send (call clif) even if the player is overweight, then the message will be triggered client side.